downstream/titles/dev-guide/master.adoc

```diff
@@ -7,7 +7,7 @@ include::attributes/attributes.adoc[]
 
 
 // Book Title
-= {PlatformName} Creator Guide
+= Ansible Automation Platform creator guide
```

Comment:
This needs to match what is in the docinfo.xml file so you should add Red Hat to the title here

Red Hat Ansible Automation Platform creator guide